luaui/Widgets/gfx_guishader.lua

```diff
@@ -68,7 +68,14 @@ end
 
 local function DrawStencilTexture(world, fullscreen)
 	--Spring.Echo("DrawStencilTexture",world, fullscreen, Spring.GetDrawFrame(), updateStencilTexture)
-	local usedStencilTex = world and stenciltex or stenciltexScreen
+	local usedStencilTex
+	if world then
+		usedStencilTex = stenciltex
+		stenciltex = nil
```

Comment:
these are different stencil textures for a reason, one fore drawing over wordsspace and one for drawing over GUI used for GUI overlappign other gui like tooltips. you dont want to nil them